community/lucene-index/src/test/java/org/neo4j/index/LegacyIndexTest.java

```diff
@@ -139,6 +142,114 @@ public void shouldThrowIllegalArgumentChangingTypeOfFieldOnNodeIndex()
     }
 
     @Test
+    public void shouldBeAbleToAddNodesAfterRemovalOfKey()
```

Comment:
This test is green w/o your changes, is that expected? How do we know that your changes solves the issue?

Reply:
This test is made green by the first commit in this PR.

Reply:
Missed the change to the db rule. All good here.